internal/api/metadata/describe_test.go

```diff
@@ -0,0 +1,40 @@
+package metadata
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func TestMetadataDescribe(test *testing.T) {
+	oldDescription := core.GetAPIDescription()
```

Comment:
Move this line to right above where you set the description (line 24).
We want those three lines to be one visual block:
"set the description just for this test".